test/powershell/Modules/PSDiagnostics/PSDiagnostics.Tests.ps1

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe -Name "PSDiagnostics cmdlets tests" -Tag "CI","RequireAdminOnWindows" {
+    Context -Name "Test for Enable-PSTrace cmdlet" {
+        it "Should enable Analytic logs for Microsoft-Windows-PowerShell." -Skip:(-not $IsWindows) {
```

Comment:
If all tests in the file is skipped on non-Windows we need use our common pattern for skipping https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#skipping-tests-in-bulk

Reply:
Ok, didn't check this doc, Thanks.